addons/medical_engine/XEH_postInit.sqf

```diff
@@ -62,3 +62,11 @@
         [QEGVAR(medical,woundReceived), [_x, "Head", _lethality, _killer, "#vehiclecrash", [HITPOINT_INDEX_HEAD,1]], _x] call CBA_fnc_targetEvent;
     } forEach (crew _vehicle);
 }, true, ["ParachuteBase"]] call CBA_fnc_addClassEventHandler;
+
+// fixes units being stuck in unconscious animation when being knocked over by a physix object
+["CAManBase", "AnimDone", {
+    params ["_unit", "_anim"];
+    if (local _unit && {_anim == "unconscious" && {lifeState _unit != "INCAPACITATED"}}) then {
+        [_unit, false] call FUNC(setUnconsciousAnim);
+    };
+}, true, [], true] call CBA_fnc_addClassEventHandler;
```

Comment:
```suggestion
}] call CBA_fnc_addClassEventHandler;
```
These are the default values, except "apply retroactively", but that is only relevant for init and init post events.